src/components/ToggleSwitch/index.tsx

```diff
@@ -169,7 +178,7 @@ const ToggleSwitch: React.FC<ToggleSwitchProps> = ({
       style={[styles.container, containerStyle]}
       activeOpacity={0.9}
       onPress={onPress}>
-      <Text style={[styles.labelLeft, styleLabelLeft]}>{labelLeft}</Text>
+      {labelLeft && <Text style={[styles.labelLeft, styleLabelLeft]}>{labelLeft}</Text>}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The conditional rendering of the `labelLeft` text component based on its existence is a good practice. This prevents unnecessary rendering and potential errors if `labelLeft` is undefined or null.
```diff
+ {labelLeft && <Text style={[styles.labelLeft, styleLabelLeft]}>{labelLeft}</Text>}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->